conf/sample.ini

```diff
@@ -110,6 +110,11 @@
 # For "sqlite3" only. cache mode setting used for connecting to the database. (private, shared)
 ;cache_mode = private
 
+################################### Datasources #########################
+[datasources]
+# Max data sources return in the result default is 5000
```

Comment:
I wouldn't document the default in a comment here, since it overlaps with the default in defaults.ini, and you risk them going out of sync. I would just use the same documentation comment for both this file and defaults.ini.
```suggestion
# Upper limit of data sources that Grafana will return
```